### PR TITLE
Online: Updated max pod memory information/ Reduce minimum pod limit/request restrictions

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/De4LjyYx/1038-2-increase-max-pod-memory-for-pro-to-3gi
https://trello.com/c/4ECkoHlp/1066-1-pro-reduce-minimum-pod-limit-request-restrictions

Preview Build:
http://file.rdu.redhat.com/~ahardin/03162018/increase-max-pod-memory-pro/dev_guide/compute_resources.html

Note the changes in the Overview and in the Core Limit Range Object Definition example.

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@abhgupta PTAL

File: admin_guide/limits.adoc [hunk 1]
```diff
@@ -84,15 +83,71 @@ specified.
...
+        memory: "4Mi" <9>
+      default:
+        cpu: "300m" <10>
+        memory: "200Mi" <11>
+      defaultRequest:
+        cpu: "200m" <12>
+        memory: "100Mi" <13>
+      maxLimitRequestRatio:
+        cpu: "10" <14>
+----
```
Comment (yasun1) [inline admin_guide/limits.adoc]: I am sorry I don't know this limtrange.yaml is for which cluster. It is different from the ones in Pro or Starter.

File: admin_guide/limits.adoc [hunk 2]
```diff
@@ -84,15 +83,71 @@ specified.
...
+<1> The name of the limit range object.
+<2> The maximum amount of CPU that a pod can request on a node across all
+containers.
+<3> The maximum amount of memory that a pod can request on a node across all
+containers. For {product-title} Starter, the maximum is `2Gi` and for
+{product-title} Pro the maximum is `3Gi`.
```
Comment (yasun1) [inline admin_guide/limits.adoc]: Till now, I checked the Starter, the memory maximum is still '1Gi'. @abhgupta, will we have the plan to increase the Max memory to 2Gi on Starter?

File: admin_guide/limits.adoc [hunk 3]
```diff
@@ -84,15 +83,71 @@ specified.
...
+<7> The maximum amount of memory that a single container in a pod can request.
+<8> The minimum amount of CPU that a single container in a pod can request.
+<9> The minimum amount of memory that a single container in a pod can request.
+<10> The default amount of CPU that a container will be limited to use if not
+specified.
+<11> The default amount of memory that a container will be limited to use if not specified.
+<12> The default amount of CPU that a container will request to use if not specified.
+<13> The default amount of memory that a container will request to use if not specified.
+<14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
+endif::[]
```
Comment (yasun1) [inline admin_guide/limits.adoc]: Due to the current limitrange.yaml is not an Online limitrange, I can not make sure whether the items need to be updated.

**Comment by abhgupta** [comment]
I would like to see this document section be generic. This should be close/similar to what exists for OCP/Dedicated. We should add specific "alert" sections for Starter/Pro limitations. The current location of those tier-specific restrictions is somewhat hidden/buried.

File: admin_guide/limits.adoc (hunk 2 above)
Comment (abhgupta) [reply admin_guide/limits.adoc]: Starter has and will continue to have a 1Gi memory limit for pods/containers.

**Comment by ahardin-rh** [comment]
@abhgupta Thanks! I made some updates. PTAL.

File: admin_guide/limits.adoc [hunk 4]
```diff
@@ -31,8 +31,15 @@ default value is applied to the resource.
 [[limit-range-def]]
 // tag::admin_limits_sample_definitions[]
 
-.Core Limit Range Object Definition
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+For {product-title} Pro, the maximum pod memory is 3Gi. The minimum pod or
+container memory that you can specify is 100Mi.
```
Comment (abhgupta) [inline admin_guide/limits.adoc]: Lets add similar info for Starter:
Starter Min Pod Memory: 200Mi
Starter Max Pod Memory: 1Gi

**Comment by ahardin-rh** [comment]
@abhgupta Thanks! I made the updated and pushed latest changes to the preview build.

**Comment by abhgupta** [comment]
LGTM

**Review by abhgupta — APPROVED** [review]

**Comment by yasun1** [comment]
LGTM

**Pull request merged**